scripts/simplify_network.py

```diff
@@ -319,10 +319,10 @@ def simplify_links(
     exclude_carriers=[],
     aggregation_strategies=dict(),
 ):
-    ## Complex multi-node links are folded into end-points
+    # Complex multi-node links are folded into end-points
     logger.info("Simplifying connected link components")
 
-    if n.links.empty:
+    if n.links.empty or "csp" in renewable_config:
```

Comment:
Is it still needed in the updated implementation?

Reply:
We no longer need this. I will revert it to the default.